src/lib.rs

```diff
@@ -91,6 +91,8 @@ pub mod hotshot {
 
         pub const QC: &str = "QC";
         pub const LEAF: &str = "LEAF";
+
+        pub const ENCODED_PUB_KEY: &str = "PUBKEY";
```

Comment:
Nit: Is there a high overhead for having the value be `ENCODED_PUB_KEY`? If not, might be worth changing for consistency with the other tags in this file.

```suggestion
        pub const ENCODED_PUB_KEY: &str = "ENCODED_PUB_KEY";
```

Reply:
Negligible overhead. My reasoning is that "ENCODED_PUB_KEY" is kind of an implementation detail, that might cause confusion for users (the point of the tag is to be a friendly, user-visible identifier). What is a `EncodedPublicKey` to a user? It's just the serialized representation of a public key.

I was also thinking maybe `STAKING_PUBKEY`. But I wasn't sure if this type is only used for staking keys. WDYT @DieracDelta ?

Reply:
Ah! In that case, this makes sense and LGTM.

I'm not 100% clear on the usage of the tag (maybe @elliedavidson or @nyospe are?). Right now I believe the encoded public key is used only for staking but in the future we might have a type level distinction? Not 100% sure.